component/input/input.css

```diff
@@ -0,0 +1,33 @@
+.input-container {
+  padding: 1rem;
+}
+input {
+  display: block;
+  padding: 0.8rem 1rem;
+  border-radius: 0.5rem;
+  border: none;
+  width: 25%;
+  margin: 0.5rem 0;
+}
+.input-error {
```

Comment:
You can add an empty line after the closing flower brackets, it increases the readability.

Reply:
Okay, Shaik. I will remember it. Thank you.